pycbc/transforms.py

```diff
@@ -483,19 +537,57 @@ def _convert_inverse(self, maps):
                              maps[parameters.spin2x], maps[parameters.spin2y])
         return self.format_output(maps, out)
 
-# list of all Conversions to/from base parameters
-to_base_converters = [
+
+class CartesianSpinToChiP(ChiPToCartesianSpin):
+    """The inverse of ChiPToCartesianSpin.
+    """
+    name = "cartesian_spin_to_chi_p"
+    inverse = ChiPToCartesianSpin
+    _inputs = inverse._outputs
+    _outputs = inverse._inputs
+    transform = inverse.inverse_transform
+    inverse_transform = inverse.transform
+    jacobian = inverse.inverse_jacobian
+    inverse_jacobian = inverse.jacobian
+
+
+ChiPToCartesianSpin.inverse = CartesianSpinToChiP
+
+
+# dictionary of all transforms
+transforms = {
```

Comment:
Because `transforms` will eventually be a superset of `standard_transforms` (now `common_cbc_transforms`). It'll be clearer once I update #1681.